AdyenActions/Components/Await/AwaitComponent.swift

```diff
@@ -7,13 +7,18 @@
 import Adyen
 import Foundation
 
+///
+internal protocol AnyAwaitableAction {
+    var paymentData: String { get }
+}
+
 /// A component that handles Await action's.
 internal protocol AnyAwaitActionHandler: ActionComponent, Cancellable {
-    func handle(_ action: AwaitAction)
+    func handle(_ action: AnyAwaitableAction)
 }
 
 /// A component that handles Await action's.
-public final class AwaitComponent: AnyAwaitActionHandler {
+public final class AwaitComponent: ActionComponent, Cancellable {
```

Comment:
So who is handling AwaitActions now? :D

Reply:
`AwaitComponent` handles `AwaitAction`, `PollingAwaitComponent` handles `AnyAwaitableAction`. this is not the best, and might change in the future.